tarantool/msgpack_ext/types/datetime.py

```diff
@@ -495,23 +569,52 @@ def __eq__(self, other):
         Datetimes are equal when underlying datetime infos are equal.
 
         :param other: Second operand.
-        :type other: :class:`~tarantool.Datetime` or
-            :class:`~pandas.Timestamp`
+        :type other: :class:`~tarantool.Datetime`
 
         :rtype: :obj:`bool`
         """
 
         if isinstance(other, Datetime):
-            return self._datetime == other._datetime
-        if isinstance(other, pandas.Timestamp):
-            return self._datetime == other
+            return self.value == other.value
         return False
 
-    def __str__(self):
-        return self._datetime.__str__()
+    def isoformat(self):
```

Comment:
Why do we need the method in the public API?

Reply:
I think it's better to hide it in a private method/function until anyone asks for it.
Up to you.

Reply:
Removed `isoformat`, moved code to `__str__` itself